bin/options.js

```diff
@@ -22,9 +22,10 @@ function getOptions() {
   try {
     var opts = fs.readFileSync(optsPath, 'utf8')
           .trim()
```

Comment:
If the string ends with `"\\ "` trim will remove the trailing space. Edge case, I admit, but these two lines should be swapped.